src/components/AddBook.js

```diff
@@ -1,8 +1,31 @@
-const AddBook = () => (
-  <div className="add-book">
-    <input type="text" placeholder="Add new book" />
-    <button type="submit" value="add-book">Add</button>
-  </div>
-);
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const [state, setState] = useState('');
+
+  const dispatch = useDispatch();
+
+  const submitBookToStore = () => {
+    const newBook = {
+      title: state,
+      author: 'Anonimus',
+      id: uuidv4(),
+    };
+
+    dispatch(addBook(newBook));
+
+    setState('');
+  };
```

Comment:
- [ ] The component responsible for adding books should be a [controlled component](https://stackoverflow.com/questions/42522515/what-are-react-controlled-components-and-uncontrolled-components) .
it should contain:
- A local state with `title`, `author` - they should have their inputs that updates the local state every time they update (`onChange`)

Reply:
Hello, @omar-labana I don't understand what do you mean when said title and author need their own inputs? do you mean I have to add a new input element for the author? or just update the state to create that as default?
I am asking because the sneak peek shows just one input for the book title.

Reply:
Hi @marilirulita, they should be updated on input change using the `onChange` listener and useState hook